backend/core/src/migration/1656362047737-user-role-updates.ts

```diff
@@ -0,0 +1,25 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class userRoleUpdates1656362047737 implements MigrationInterface {
+  name = "userRoleUpdates1656362047737"
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+            UPDATE user_roles
+            SET 
+                is_admin = true,
+                is_partner = false,
+                is_jurisdictional_admin = false
+            WHERE is_admin = true AND (is_jurisdictional_admin = true OR is_partner = true);
+
+            UPDATE user_roles
```

Comment:
The first update will always set jurisdictional admin to false where is_jurisdictional_admin is true, and the second is looking for records where is_jurisdiction_admin is true, which I think due to the first update will never be the case - might be missing something